README.md

```diff
@@ -1,8 +1,53 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nisrein Hinnawi . Jarvis Consulting
 
+I completed my Bachelor's degree in Computer Science from Jordan University of Science and Technology, and a Diploma degree in Computer Programming from Seneca College with honor, and I have a. During my studies, I have grown immense interest and enthusiasm in the field of research and development due to its innovative nature. Thinking out of the box, implementing new ideas, and finally developing those ideas into innovations are some of the main reasons which attract me into this field. I have the critical skills and knowledge which the market is looking for. I can work on different project platforms built with Java, C/C++, or C#. I have the ability to adapt and be an active team member in a short time, learn quickly new techniques and software as I have a strong and thorough understanding of programming languages, and meet stringent deadlines if required.
+
+## Skills
+
+**Proficient:** Java, C/C++, JavaScript, HTML/CSS, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker, IntelliJ, Eclipse, Visual  Studio, Visual Studio Code, .NET(C#)
+
+**Familiar:** Java SDK, Angular.js, Xamarin, IBM Zeus, Hadoop framework
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NisreinHinnawi](https://github.com/jarviscanada/jarvis_data_eng_NisreinHinnawi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NisreinHinnawi/tree/master/linux_sql)]: Developed a Linux monitoring agent that helps the user record and monitor hardware specifications and usage information (e.g. CPU, RAM, Disk, etc.) for a Linux cluster in real-time and stores the data in a PostgreSQL database. The program was developed in Bash Scripts and tested on a virtual machine that uses CentOS 7 distribution, and the Docker container was used to provision the PostgreSQL database.
```

Comment:
**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NisreinHinnawi/tree/master/linux_sql)]: Developed a Linux monitoring agent that helps the user record and monitors hardware specifications and usage information (e.g. CPU, RAM, Disk, etc.) for a Linux cluster in real-time and stores the data in a PostgreSQL database. The program was developed in Bash Scripts and tested on a virtual machine that uses CentOS 7 distribution, and the Docker container was used to provision the PostgreSQL database.